Chave do CFe gera o pos.order name

leorochael opened the pull request:
[ADD] Chave do CFe gera o pos.order name

- é criada a associação entre o nome da loja configurado no pos.config e
 os valores do número do sat e o número da venda retirados da chave do
 cupom fiscal após o retorno da SEFAZ.

mileo commented on l10n_br_pos/models/pos_order.py:
```diff
@@ -99,6 +99,13 @@ def write(self, vals):
+        pos_config = order.session_id.config_id
+        if pos_config.iface_sat_via_proxy:
+            sequence = pos_config.sequence_id
+            order.name = sequence._interpolate_value("%s / %s" % (
```
Recomendo o uso da seguinte classe 

https://github.com/base4sistemas/satcomum/blob/master/satcomum/ersat.py#L85

leorochael committed:
fixup! [ADD] Informacoes adicionais no cupom fiscal

[ADD] Chave do CFe gera o pos.order name

- é criada a associação entre o nome da loja configurado no pos.config e
 os valores do número do sat e o número da venda retirados da chave do
 cupom fiscal após o retorno da SEFAZ.
